packages/react-theme-provider/README.md

```diff
@@ -135,3 +135,99 @@ const Foo = props => {
   return <div className={classes.root} />;
 };
 ```
+
+## How does this change other existing ways of theming Fluent UI components?
+
+### Customizer
+
+`Customizer` is now deprecated and you should replace it with `ThemeProvider`.
```

Comment:
It would be good to have a before/after code example showing how ThemeProvider can take in "components" for scoped settings.

Reply:
i have that below under `#### Replace scopedSettings prop`